tomcat/src/main/java/org/apache/coyote/RequestMapping.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.coyote;
+
+import org.apache.file.ResourceHandler;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RequestMapping {
+    private static final Map<String, Controller> controllers = new HashMap<>();
+    private static Controller defaultController;
```

Comment:
```suggestion
    private static final Map<String, Controller> controllers = new HashMap<>();
    
    private static Controller defaultController;
```

상수와 상태 값은 구분지어주면 좋습니다.